references/threat-protect/test/integration/features/threat-protect.json.feature

```diff
@@ -0,0 +1,30 @@
+@JSONEndpoint
+Feature:
+  As an API Developer
+  I want to generate a JSON threat
+  So that I can verify JSON threat protection
+
+  Scenario: Generate a valid and conform JSON content and verify it
+    When I GET /json?width=`width`&depth=`depth`&length=`length`&height=`height`
+    Then response code should be 200
+    And response body should be valid json
+
+  Scenario: Generate a JSON threat (depth attack) and reject it
+    When I GET /json?width=`width`&depth=20&length=`length`&height=`height`
+    Then response code should be 500
```

Comment:
Up for discussion. I think this should be a 4xx client error and not a server error.

Reply:
Yes good point. I modified to return status code 400 and message "Bad Request"